File: GildedRose/src/com/gildedrose/items/LegendaryItem.java
package com.gildedrose.items;

public class LegendaryItem extends ItemDecorator {
	
	// Constructors
	protected LegendaryItem(Item itemToDecorate)
	{
		super(itemToDecorate);
	}

	// Protected methods
	@Override
	protected void updateSellIn() { }

	@Override
	protected void updateQuality() { }

}

File: GildedRose.Net/Items/LegendaryItem.cs
namespace GildedRose.Items
{
    public class LegendaryItem : ItemDecorator
    {
	
		// Constructors
		internal LegendaryItem(Item itemToDecorate) : base(itemToDecorate) { }
	
		// Protected methods
		protected override void UpdateSellIn() { }
	
		protected override void UpdateQuality() { }
	
	}
}